src/mjolnir/ingest_transit.cc

```diff
@@ -601,12 +602,15 @@ bool write_stop_pair(
 }
 
 // read routes data from feed
-void write_routes(Transit& tile, const tile_transit_info_t& tile_info, feed_cache_t& feeds) {
+std::unordered_map<feed_object_t, size_t>
+write_routes(Transit& tile, const tile_transit_info_t& tile_info, feed_cache_t& feeds) {
 
   const auto& tile_routeIds = tile_info.routes;
 
-  // loop through all stops inside the tile
+  std::unordered_map<feed_object_t, size_t> routes_ids;
```

Comment:
we introduce this to align well with `convert_transit`, where we're adding a tile's `TransitRoute` based on the index in the transit tile's PBF